packages/gp2-components/src/organisms/ProjectCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import { format } from 'date-fns';
+import {
+  Card,
+  crossQuery,
+  ExternalLink,
+  lead,
+  LinkHeadline,
+  pixels,
+  utils,
+} from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import dateIcon from '../icons/date-icon';
+import usersIcon from '../icons/users-icon';
+import IconWithLabel from '../molecules/IconWithLabel';
+import ProjectStatus, { statusStyles } from '../molecules/ProjectStatus';
+import colors from '../templates/colors';
+
+const { getCounterString } = utils;
+const { rem } = pixels;
+
+type ProjectCardProps = Pick<
+  gp2Model.ProjectResponse,
+  | 'title'
+  | 'status'
+  | 'startDate'
+  | 'endDate'
+  | 'members'
+  | 'id'
+  | 'projectProposalUrl'
+>;
+
+const calcDuration = (start: string, end?: string) => {
+  if (!end) return '';
+  const startDate = new Date(start);
+  const endDate = new Date(end);
+
+  const monthDuration =
+    12 * (endDate.getFullYear() - startDate.getFullYear()) +
+    (endDate.getMonth() - startDate.getMonth());
+  return `(${monthDuration} mos)`;
+};
```

Comment:
Instead of rolling our own we could use date-fns:
```
import formatDistance from "date-fns/formatDistanceStrict"

formatDistance(new Date(2020, 7, 6), new Date(2021,12,28), { unit: 'month'})

```

Reply:
For example I'm not sure how our code handles sub-months. 

i.e. 

The month difference between:

2020-07-22 and 2021-12-28 is 17 months
2020-07-21 and 2021-12-28 is 18 months

https://codesandbox.io/s/priceless-mahavira-p9l036?file=/src/App.tsx

Not sure we need to be so accurate as we are no displaying the day of the month on the frontend?